sigle/src/modules/editor/ShortcutsDialog.tsx

```diff
@@ -0,0 +1,448 @@
+import {
+  Dialog,
+  DialogContent,
+  DialogTrigger,
+  IconButton,
+  Tabs,
+  TabsContent,
+  TabsList,
+  TabsTrigger,
+} from '../../ui';
+import { styled } from '../../stitches.config';
+import { KeyboardIcon } from '@radix-ui/react-icons';
+import { HintsCarousel } from './HintsCarousel';
+
+const StyledTable = styled('table', {
+  width: '100%',
+});
+
+const Tr = styled('tr', {
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  gap: '$5',
+  py: '$4',
+  mb: '$2',
+  boxShadow: '0 1px 0 0 $colors$gray6',
+});
+
+const Th = styled('th', {
+  fontSize: '$2',
+  fontWeight: 400,
+  color: '$gray11',
+  flex: 1,
+  maxWidth: '30%',
+  textAlign: 'left',
+});
+
+const Td = styled('td', {
+  display: 'flex',
+  gap: '$2',
+  maxWidth: '30%',
+  flex: 1,
+  textAlign: 'left',
+});
+
+interface TableProps {
+  children: React.ReactNode;
+}
+
+const Table = ({ children }: TableProps) => (
+  <StyledTable>
+    <thead>
+      <Tr css={{ boxShadow: '0 1px 0 0 $colors$gray12' }}>
+        <Th scope="col">Command</Th>
+        <Th scope="col">Windows/Linux</Th>
+        <Th scope="col">macOS</Th>
+      </Tr>
+    </thead>
+    <tbody>{children}</tbody>
+  </StyledTable>
+);
+
+const Kbd = styled('kbd', {
+  all: 'unset',
+  display: 'grid',
+  placeItems: 'center',
+  br: '$1',
+  backgroundColor: '$gray11',
+  color: '$gray1',
+  p: '$1',
+  fontFamily: '$monaco',
+  fontSize: '$2',
+});
+
+export const ShortcutsDialog = () => {
+  return (
+    <Dialog>
+      <DialogTrigger asChild>
+        <IconButton aria-label="Open keyboard shortcuts and hints">
+          <KeyboardIcon />
+        </IconButton>
+      </DialogTrigger>
+      <DialogContent
+        css={{
+          backgroundColor: '$gray1',
+          maxWidth: '760px',
+          px: '$8',
+          py: '$8',
+        }}
+      >
+        <Tabs defaultValue="shortcuts">
+          <TabsList aria-label="Find keyboard shortcuts and hints">
+            <TabsTrigger value="shortcuts">Keyboard Shortcuts</TabsTrigger>
+            <TabsTrigger value="hints">Hints</TabsTrigger>
+          </TabsList>
+
+          <TabsContent css={{ height: 600 }} value="shortcuts">
+            <Tabs defaultValue="essentials">
+              <TabsList css={{ mb: '$4' }} aria-label="Explore shortcut types">
+                <TabsTrigger value="essentials">Essentials</TabsTrigger>
+                <TabsTrigger value="text-formatting">
+                  Text Formatting
+                </TabsTrigger>
+                <TabsTrigger value="p-formatting">
+                  Paragraph Formatting
+                </TabsTrigger>
+                <TabsTrigger value="selection">Text Selection</TabsTrigger>
+              </TabsList>
+              <TabsContent value="essentials">
+                <Table>
+                  <Tr>
+                    <Th scope="row">Copy</Th>
```

Comment:
I feel like all this part could be simplified a lot. By doing a javascript object, adding new items later will be easier too :)

smth like :
```tsx
const array = [{
  action: 'Copy',
  command: '...'
}, // ....rest
];

array.map(() => (
<Tr>
<Th scope="row">{array.action}</Th>
// ..... rest
))
```

Reply:
Ahh yes of course, this makes much more sense 🤦‍♂️  Thanks for the suggestion! 

Do you think it would best to store the array in its own config file?

Reply:
As it's the only place where we use it it's fine to have it in the same file